src/components/screens/login/login.js

```diff
@@ -46,6 +47,8 @@ const Login = ({
   const [isRecoveryPhraseMode, setIsRecoveryPhrase] = useState(false);
   const [derivationPath, setDerivationPath] = useState(defaultDerivationPath);
   const [passphrase, setPass] = useState({ value: '', isValid: false });
+  const canHWSignIn = true;
```

Comment:
Is this planned to be replaced by a logic? Cause else you can get rid of this variable and the corresponding ternary.

Reply:
I think we can replace this, I just added this to be easily testable, but we can remove this 👍🏼